Antidote/ToxListener.m

```diff
@@ -124,6 +126,24 @@ - (void)controllerDidChangeContent:(RBQFetchedResultsController *)controller
     }
 }
 
+#pragma mark - SubmanagerCalls delegate
+- (void)callSubmanager:(OCTSubmanagerCalls *)callSubmanager receiveCall:(OCTCall *)call audioEnabled:(BOOL)audioEnabled videoEnabled:(BOOL)videoEnabled
+{
+    AppDelegate *delegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
+    UIViewController *visibleVC = [delegate visibleViewController];
+
+    if ([visibleVC isKindOfClass:[CallViewController class]]) {
+        // User is in a middle of call, send some kind of notification?
```

Comment:
Mannol said yes to both questions. I guess then we need to change that in objcTox as well then. Though it shouldn't be that big of a change. Will need to find a way to get a third device to test this :sweat_smile: